Maths/CartesianProduct.js

```diff
@@ -8,13 +8,13 @@
 const cartesianProduct = (setA, setB) => {
   // Check if input sets are not empty.
   if (!setA || !setB || !setA.length || !setB.length) {
-    return null
+    return {}
```

Comment:
hi @appgurueu, this looks ok?

Reply:
An empty dict.? Shouldn't this be an empty array `[]`?

Reply:
hmm, yes